lists/lists.css

```
@@ -0,0 +1,83 @@
+@import url(/CSS/nav.css);
+
+
+.main-container {
+
+    display: flex;
+    flex-direction: column;
+    max-width: 83%;
+    margin-top: 2rem;
+    position: relative;
+    padding-left: 19rem;
+}
+
+h3 {
```

Here, can we use a utility class like .h3 instead of directly using an element selector:

```suggestion
.h3 {
```

---

done, thank you!